File: Hontail_Back/src/main/java/com/hontail/back/ingredient/service/IngredientService.java
package com.hontail.back.ingredient.service;

import com.hontail.back.cocktail.dto.CocktailSummaryDto;
import com.hontail.back.db.entity.Ingredient;
import com.hontail.back.ingredient.dto.IngredientDto;
import com.hontail.back.ingredient.dto.IngredientSearchDto;

import java.util.List;

public interface IngredientService {
    public List<IngredientDto> getAllIngredients();

    public List<CocktailSummaryDto> getRecommendedCocktails(Integer userId, List<String> ingredients);
}

File: Hontail/app/src/main/java/com/hontail/data/remote/IngredientService.kt
package com.hontail.data.remote

import com.hontail.data.model.dto.IngredientsTable
import retrofit2.http.GET

interface IngredientService {

    // 재료 모든 데이터 가져오기.
    @GET("api/ingredients")
    suspend fun getAllIngredients(): List<IngredientsTable>
}